integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -279,6 +280,7 @@ public final void before() throws Exception {
     mFsMaster =
         mLocalTachyonClusterResource.get().getMaster().getInternalMaster().getFileSystemMaster();
     mMasterTachyonConf = mLocalTachyonClusterResource.get().getMasterTachyonConf();
+    TTLBucket.setTTlIntervalMs(mMasterTachyonConf.getLong(Constants.MASTER_TTLCHECKER_INTERVAL_MS));
```

Comment:
Couple of points here:

1) I think it does not make sense to expose an API that can create an inconsistent bucket state in production (changing the value of the interval while there are non-empty buckets seems to have the effect); the setter is only used in tests, which suggests we should use WhiteBox instead

2) In some tests, we attempt to set the bucket interval through the setter, while in other tests through the `Constants.MASTER_TTLCHECKER_INTERVAL_MS` value in TachyonConf. This is inconsistent and error prone. For instance, what happens if the TfsShell test runs after the FileSystemMasterIntergration test? Will the TfsShell test use incorrect value too? The root of this problem is that we are "caching" a configuration value in a static variable. What if we got rid of it and made the stateic getter consult TachyonConf every time?